pull-kueue-verify-main

<!--
Please use this template for reporting flaky tests.
Links to specific failures in Prow are appreciated.
-->

**Which test is flaking?**:

pull-kueue-verify-main

**Link to failed CI job or steps to reproduce locally**:
https://prow.k8s.io/view/gs/kubernetes-ci-logs/pr-logs/pull/kubernetes-sigs_kueue/11872/pull-kueue-verify-main/2067271243471523840

**Failure message or logs**:
```
 make[1]: Entering directory '/home/prow/go/src/sigs.k8s.io/kueue'
cd hack/testing/kal-linter; /home/prow/go/src/sigs.k8s.io/kueue/bin/golangci-lint custom; mv bin/golangci-lint-kube-api-linter /home/prow/go/src/sigs.k8s.io/kueue/bin
level=warning msg="go: downloading sigs.k8s.io/kube-api-linter v0.0.0-20250908163129-65a570bd22aa\ngo: sigs.k8s.io/kube-api-linter@v0.0.0-20250908163129-65a570bd22aa: verifying module: sigs.k8s.io/kube-api-linter@v0.0.0-20250908163129-65a570bd22aa: Get \"https://sum.golang.org/lookup/sigs.k8s.io/kube-api-linter@v0.0.0-20250908163129-65a570bd22aa\": net/http: TLS handshake timeout\n"
Error: build process: add to go.mod: go get sigs.k8s.io/kube-api-linter@v0.0.0-20250908163129-65a570bd22aa: exit status 1
The command is terminated due to an error: build process: add to go.mod: go get sigs.k8s.io/kube-api-linter@v0.0.0-20250908163129-65a570bd22aa: exit status 1
make[1]: *** [Makefile-deps.mk:84: golangci-lint-kal] Error 3

```

**Anything else we need to know?**:

--- a/Makefile-deps.mk
+++ b/Makefile-deps.mk
@@ -81,7 +81,7 @@ golangci-lint: ## Download golangci-lint locally if necessary.
 
 .PHONY: golangci-lint-kal
 golangci-lint-kal: golangci-lint ## Build golangci-lint-kal from custom configuration.
-	cd hack/testing/kal-linter; $(GOLANGCI_LINT) custom; mv bin/golangci-lint-kube-api-linter $(BIN_DIR)
+	cd hack/testing/kal-linter; $(NETWORK_INSTALL_RETRY) $(GOLANGCI_LINT) custom; mv bin/golangci-lint-kube-api-linter $(BIN_DIR)
 
 .PHONY: controller-gen
 controller-gen: gomod-download-tools ## Download controller-gen locally if necessary.

